docs/src/snforge-advanced-features/storage-cheatcodes.md

```diff
@@ -163,7 +169,7 @@ fn store_in_complex_mapping() {
 
 ## Example with `storage_address_from_base`
 
-This example uses `storage_address_from_base` with `address` function of the [storage variable](https://book.cairo-lang.org/ch14-01-contract-storage.html#addresses-of-storage-variables).
+This example uses `storage_address_from_base` with `address` function of the storage variable.
```

Comment:
```suggestion
This example uses `storage_address_from_base` with `address` function of the [storage variable](https://book.cairo-lang.org/ch14-01-00-contract-storage.html#addresses-of-storage-variables).
```

maybe ?

Reply:
I considered it but the API is different from what's presented in the guide, so it doesn't seem relevant anymore